gulp-tasks/workbox:generate-contributors.js

```diff
@@ -0,0 +1,94 @@
+const GitHubApi = require('github');
+const fs = require('fs-extra');
+const path = require('path');
+const gulp = require('gulp');
+
+const MAX_COLLABORATORS = 20;
+
+function getContributorHTML(contributorStats) {
+  const contributor = contributorStats.author;
+  var html = `
+<div class="wb-contrib">
+  <a class="wb-contrib__img-link" href="${contributor.url}" target="_blank">
+    <img class="wb-contrib__image" src="${contributor.avatar_url}">
+  </a>
+  <br />
+  <a class="wb-contrib__link" href="${contributor.url}" target="_blank">
+    ${contributor.login}
+  </a>
+</div>
+`;
+  return html;
+}
+
+gulp.task(`workbox:generate-contributors`, function() {
+  if (!process.env.GITHUB_TOKEN) {
```

Comment:
Let's change this to the same way that the youtube token works, that way not everyone needs to have this setup.